services/status/api.go

```diff
@@ -54,14 +54,16 @@ type SignupRequest struct {
 
 // SignupResponse : json response returned by status_signup.
 type SignupResponse struct {
-	Address  string `json:"address"`
-	Pubkey   string `json:"pubkey"`
-	Mnemonic string `json:"mnemonic"`
+	Address     string `json:"address"`
```

Comment:
Do we want to rename this to `WalletAddress` and just leave the JSON identifier as `address`?
```suggestion
	WalletAddress     string `json:"address"`
```